blast/2.13.0/README.md

```diff
@@ -0,0 +1,18 @@
+# blast+ container
+
+Main tools:
+- [blast+](https://blast.ncbi.nlm.nih.gov/Blast.cgi?PAGE_TYPE=BlastDocs&DOC_TYPE=Download)
+
```

Comment:
Might be worth mentioning that ncbi-blast+ is a suite of tools. For example I had no idea it included `dustmasker` and `segmasker` which is optional but recommended for kraken2 database building. You could list all the executables included:
```
ls /ncbi-blast-2.13.0+/bin/
blast_formatter      blastdb_aliastool  blastn      blastx                      deltablast             legacy_blast.pl  makeprofiledb  rpstblastn  tblastn_vdb        windowmasker
blast_formatter_vdb  blastdbcheck       blastn_vdb  cleanup-blastdb-volumes.py  dustmasker             makeblastdb      psiblast       segmasker   tblastx
blast_vdb_cmd        blastdbcmd         blastp      convert2blastmask           get_species_taxids.sh  makembindex      rpsblast       tblastn     update_blastdb.pl
```

Reply:
I think this is a good idea, especially since two of the tools won't work in this container 
 - get_species_taxids.sh (requires E-direct)
 - update_blastdb.pl     (requires perl)
I could add these two in, but since they aren't tools I normally use I wasn't sure how to test them.